.github/workflows/build-and-test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup - Xcode
         uses: maxim-lobanov/setup-xcode@v1
         with:
-          xcode-version: "13.0"
+          xcode-version: "13.3.1"
```

Comment:
Had to update this to ensure the UI tests involving Apple Pay work, locally my simulator uses iOS 15.4 whereas the version used previously here was 15.0 (now 15.4 with this change).

The tests were failing, I suspect due to the Apple Pay sheet appearing slightly different in that version, and therefore the UI tests couldn't find the buttons they are expecting